5.5/test/run

```diff
@@ -19,13 +19,18 @@ cleanup() {
 
 		echo "Stopping and removing container $CONTAINER..."
 		docker stop $CONTAINER
+		exit_status=$(docker inspect -f '{{.State.ExitCode}}' $CONTAINER)
+		if [ "$exit_status" != "0" ]; then
+			echo "Dumping logs for $CONTAINER"
+			docker logs $CONTAINER
+		fi
```

Comment:
Hmm. Well, this is cleanup, so if it exited with non-zero status, the cleanup was probably called by trap and will exit with non-zero status anyway. I'm not sure what would happen if I'd put another exit there..

I think a situation where exit is normal but container exits with error shouldn't happen.